### PR TITLE
feat: Add Webpack 4 support.

### PR DESCRIPTION
**Pull request by natedanner**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
     "ava": "^0.19.0",
     "babel-eslint": "^7.1.1",
     "dustjs-linkedin": "^2.7.5",
-    "eslint": "^3.17.1",
+    "eslint": "^5.15.3",
     "husky": "^0.13.2",
     "memory-fs": "^0.4.1",
-    "webpack": "^2.2.1"
+    "webpack": "^4.29.6"
```
Comment (avaly) [inline package.json]: Please add `webpack: >=4.0.0` to the `peerDependencies` list

Comment (natedanner) [reply package.json]: The loader still works on 2 and 3. Ok with setting `"webpack": ">=2.0.0"`?

Comment (natedanner) [reply package.json]: peerDependencies pushed up. Thanks!

**Comment by avaly** [comment]
@natedanner this PR broke tests on master: https://travis-ci.org/avaly/dust-loader/jobs/508900516

I should have enabled Travis for PRs before :man_facepalming: 	

I'd love if you could send an updated PR. Thanks!

**Comment by natedanner** [comment]
Doh. I'll get that fixed up. Looks like just a dependency needs updating.

**Pull request merged**